app/advancedfilters/advancedFiltersModal.html

```diff
@@ -311,11 +308,10 @@
 
   </div><!-- End of my tabs -->
 
-  <div>
+  <div class="advancedFiltersActionButtons">
```

Comment:
It would be nicer to add an <hr> above the buttons rather than style this div. Also this class name is unsemantic.